sdk/rust/README.md

```diff
@@ -0,0 +1,88 @@
+# PVRecorder Rust Binding
```

Comment:
I know it's pedantic, but we usually write Pv in camel case: PVRecorder >>> PvRecorder